README.md

```diff
@@ -64,10 +65,10 @@ cloudasset.googleapis.com
 
 | Name | Version |
 |------|---------|
-| <a name="provider_google"></a> [google](#provider\_google) | >= 4.4.0, < 5.0.0 |
-| <a name="provider_lacework"></a> [lacework](#provider\_lacework) | ~> 0.2 |
-| <a name="provider_random"></a> [random](#provider\_random) | n/a |
-| <a name="provider_time"></a> [time](#provider\_time) | ~> 0.6 |
+| <a name="provider_google"></a> [google](#provider\_google) | 4.28.0 |
```

Comment:
I would just double check these version constraints. From the previous pinning we were constrained to >= 4.4.0, < 5.0.0

Reply:
Reverted commit - switching back to the original hand edited README. Not sure how to specify the version constraints in code, but I guess code and README are not in sync.